app/presenters/track.rb

```diff
@@ -47,12 +47,12 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        template = File.read("./x/docs/md/track/BETTER.md")
+        template = File.read(shared_template_filepath("BETTER"))
         populate_template_with_trackler_data(template).gsub('TOPIC', topic_key.to_s.upcase)
       end
 
       def fallback_topic_content(topic_key)
```

Comment:
You make a good point, I think I'd become so focused on those two methods that I lost track of the context.